app/views/new_document/choose_document_type.html.erb

```diff
@@ -13,7 +13,7 @@
           {
             value: document_type.id,
             text: document_type.label,
-            hint_text: document_type.description,
+            hint_text: t("document_types.#{document_type.id}.description"),
```

Comment:
It's a shame this caused such an issue with the tests. While the factory solution is pretty neat, could we think about making the description optional? I agree that's not exactly a refactoring, but we [have a precedent for this sort of thing](https://github.com/alphagov/content-publisher/blob/bd10dd6c488102b53ba2ce1fa85fa923d16bdd46/app/models/document_type.rb#L24), where it's beneficial for the tests. The upcoming work on testing formats should ensure that any "real" format do provide a description, so I don't see a great risk of tweaking the behaviour here to make things easier.

Reply:
Have now made it optional (locally -will push soon).